src/appmixer/stripe/auth.js

```
@@ -0,0 +1,47 @@
+                headers: {
+                    Authorization: `Bearer ${apiKey}`
+                }
+            });
+
+            return response.data;
+        },
+
+        async requestProfileInfo(context) {
+            const data = await this.getUserProfile(context);
+
+            if (data && data.fullName) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)